test/integration/datapath/datapath_linux_test.go

```diff
@@ -163,11 +174,15 @@ func setupLinuxEnvironment(t *testing.T) {
 		if err != nil {
 			require.NoError(t, err)
 		}
-		t.Log("Successfully created customer linux pods")
+
+		if *isDualStack {
+			t.Log("Successfully created customer dualstack Linux pods")
+		} else {
+			t.Log("Successfully created customer singlestack Linux pods")
+		}
 	} else {
-		t.Log("Checking for pods to be running state")
-		err = k8sutils.WaitForPodsRunning(ctx, clientset, *podNamespace, podLabelSelector)
-		if err != nil {
+		// delete namespace
+		if err := k8sutils.MustDeleteNamespace(ctx, clientset, *podNamespace); err != nil {
 			require.NoError(t, err)
```

Comment:
Need to ensure the test does not run after the namespace is deleted as it will always fail or change logic.

Reply:
Yes we should delete namespace and stop test here.